utils/parametric/protos/apm_test_client.proto

```diff
@@ -211,6 +264,35 @@ message OtelSetAttributesArgs {
 }
 message OtelSetAttributesReturn {}
 
+message OtelCurrentSpanArgs {}
+message OtelCurrentSpanReturn {
+  uint64 span_id = 1;
+  uint64 trace_id = 2;
+  uint64 parent_id = 3;
+}
+
+message OtelGetAttributeArgs {
+  uint64 span_id = 1;
+  string key = 2;
+}
+message OtelGetAttributeReturn {
+  AttrVal value = 1;
+}
```

Comment:
OTel attributes support different types of arguments.
We might need to test them too.